pkg/virt-controller/watch/migration.go

```diff
@@ -176,6 +176,10 @@ func (c *MigrationController) execute(key string) error {
 
 	var syncErr error
 
+	if migration.DeletionTimestamp != nil && !migration.IsFinal() {
+		migration.Status.Phase = virtv1.MigrationAborting
```

Comment:
couple of things here. 

1. we can't modify the migration object directly here because it comes from the cache. It would need to be deepcopied.

2. It might be more appropriate for this logic to live in the updateStatus() function. that's generally where the status section is altered.

Reply:
done.